### ISSUE TITLE
Regression: Bat Rider Model

### ISSUE THREAD

**Issue by SuzumiK** [ZhengPeiRu21/mod-individual-progression#280]
Bat riders were previously shown as silithid (#93)

The offending lines have since been re-introduced into the relevant .sql file

https://github.com/ZhengPeiRu21/mod-individual-progression/blame/200e153650fd7ceddf4f192c920a81c39873b949/sql/world/base/vanilla_creature_values.sql#L21395

Bat riders therefore once again appear as silithid models.

**Comment by Grimfeather** [comment]
+1 , can confirm.

temporary fix to undo the changes and change the models back to normal: 


/*  Gryphon Rider Guard  */
UPDATE `creature_template` SET `modelid1` = 17202, `minlevel` = 70, `maxlevel` = 70, `speed_run` = 2.85714, `detection_range` = 20, `DamageModifier` = 4.6, `HealthModifier` = 2 WHERE `entry`=15241;

/*  Bat Rider Guard  */
UPDATE `creature_template` SET `modelid1` = 15303, `minlevel` = 70, `maxlevel` = 70, `speed_walk` = 2.8, `speed_run` = 2.57143, `detection_range` = 20, `DamageModifier` = 2, `HealthModifier` = 2 WHERE `entry`=15242;

**Comment by ZhengPeiRu21** [comment]
My apologies, I didn't properly add this exclusion to my scripts so the issue re-appeared when I re-ran the comparison to get the latest Vmangos changes. This regression has been resolved in https://github.com/ZhengPeiRu21/mod-individual-progression/commit/351aa098732d9d8e5f7b968c41e4fe6a6061d179, and I have made the adjustments to the comparison script that should prevent it from popping up again. Thanks for reporting the issue!
